RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -6725,7 +6724,6 @@
 				88A543E12C37A4820039C6A5 /* TemplateView+MultiTier.swift in Sources */,
 				3523048A2D0A0F30003971A4 /* ErrorView.swift in Sources */,
 				887A60B72C1D037000E1A461 /* WatchTemplateView.swift in Sources */,
-				4D3BA4CD2D37D15E00668AFC /* PaywallTimelineComponent.swift in Sources */,
```

Comment:
Just double checking we want this removed from here, I see in the other change it was duplicated, but I don't see `PaywallTimelineComponent` in this section?

Reply:
Yeah yeah! It was added to both `RevenueCat` and `RevenueCatUI` targets which was wrong 😅 So I removed the `RevenueCatUI` one since this file is part of the offerings response codable stuff 

But thanks for checking 🙏